test/dispatcher.js

```diff
@@ -1,7 +1,19 @@
+define([ 'intern!tdd',
+        'intern/chai!',
```

Comment:
hmmmm...my local looks like this:
define([ 'intern!tdd',
        'intern/chai!',
        'intern/chai!expect',
        '../dist/PEP'
    ],

'git status' says there is nothing to commit, perhaps my rebase pick/squash squashed the wrong thing, but strange that it is still local.